### DISCUSSION TITLE
ASUS Tinker Board

### DISCUSSION THREAD

**Discussion by Tasm-Devil** [adafruit/Adafruit_Blinka#1057] [New Board Requests]
Hey there,

can you please add support for the tinker board. It uses as SoC a Rockchip RK3288 

I use some of them in my course and it would be awesome to mess around with some sensors. 

**Thank you very much.**

**Comment by iomari** [comment]
Yes, Tinkerboard support would be very welcome.

**Comment by makermelissa** [comment]
Thanks for requesting this. No ETA at the moment, but I agree this would be a good one to add.

**Comment by ladyada** [comment]
@BlitzCityDIY wanna do this one with @makermelissa 's help?

**Comment by BlitzCityDIY** [comment]
funny story- i had actually forked recently with the intent to try porting following the guide so I would definitely be interested.

**Comment by makermelissa** [comment]
Please let me know if you see anything that you see could be improved in the guide. Thanks!

**Comment by jfs10** [comment]
Tinkerboard support would be great.

**Comment by mattsmithuk** [comment]
Would also love Tinkerboard support!

**Comment by tioguerra** [comment]
Please add Tinker Board support.

**Comment by ladyada** [comment]
@BlitzCityDIY hey do ya need help with this?

**Comment by BlitzCityDIY** [comment]
sorry somehow missed this ping. i hadn't really had time recently but will try to work on this now :)

**Comment by ladyada** [comment]
all good - we figured ya knew the hardware best!

**Comment by ElephantsFoot** [comment]
It would be great to have a Tinkerboard implementation! 😊

**Comment by earlthesquirrel** [comment]
Any progress?  I have a Tinker board that I haven't found a good use for yet, but definitely could use for CircuitPython for some projects that I have in flight
